### ISSUE TITLE
Error when try to get properties or subjects from document

### ISSUE THREAD

**Issue by jairo-campos-JD** [solid/query-ldflex#31]
When I try to get the properties from my webId I'm getting this error ```Error: The JSON-LD context cannot expand the 'properties' property```

My code is the following: 

```const user = await data['https://jairocr.inrupt.net/profile/card#me']
    for await (let subject of user.properties) {
        console.log(subject)
    }
```

**Comment by RubenVerborgh** [comment]
This is the exact error message you would get when using `@solid/query-ldflex` with `ldflex` versions lower than `2.3.0`.

I have tried your above snippet with `ldflex@2.3.0`, `ldflex@2.4.0`, `ldflex@2.5.0` and they all work (but it broke, as expected, with `ldflex@2.2.0`). I have also run it on the playground, and it works there as well: https://solid.github.io/ldflex-playground/#.user.properties

In short, I have a strong suspicion that somehow an old `ldflex` version is present. If you think that's not the case, I will probably need a zip of your entire folder, including `node_modules`.

**Comment by jairo-campos-JD** [comment]
@rubensworks yes you are right I was using the correct version on the library but not into the generator.

Thank you !!!
